Plugin/OrderPlugin.php

```diff
@@ -113,6 +121,10 @@ public function beforeSetStatus(Order $subject, $status)
             return [Order::STATE_PROCESSING];
         }
         
+        if (!$this->featureSwitches->isDisallowOrderStatusUpdatedBack()) {
```

Comment:
Could you please refactor this to have false as default value? So we can set true to specific merchant when needed